sleap_nn/data/confidence_maps.py

```diff
@@ -14,7 +14,7 @@ def make_confmaps(
     """Make confidence maps from a batch of points for multiple instances.
 
     Args:
-        points_batch: A tensor of points of shape `(batch_size, n_nodes, 2)` and dtype `torch.float32` where
+        points_batch: A tensor of points of shape `(samples, n_nodes, 2)` and dtype `torch.float32` where
```

Comment:
Consistency: use the `n_` prefix everywhere or nowhere, e.g., either `(n_samples, n_nodes, 2)` (preferred) or `(samples, nodes, 2)`.